mantidproject/mantid
# Add ability to sort a workspace by size in workspace widget

Re #22809

**Description of work.**
I have added the ability to sort all workspaces that are currently loaded by sorting by Memory Size as per issue #22809 
**Report to:** Pascal Manuel  

**To test:**

- Load multiple workspaces of differing sizes that are plainly obvious
- In the workspace widget `Sort->Size`
- It should sort by the size now

Fixes #22809 . 

Could it be advised on whether or not it needs to be added as like a minor feature on release notes.

Does this update require release notes?


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
